sklearn/feature_selection/univariate_selection.py

```diff
@@ -390,13 +576,17 @@ class SelectPercentile(_BaseFilter):
     f_classif: ANOVA F-value between label/feature for classification tasks.
     mutual_info_classif: Mutual information for a discrete target.
     chi2: Chi-squared stats of non-negative features for classification tasks.
+    info_gain: Information Gain of features for classification tasks.
```

Comment:
IMO, this is getting silly, especially now that we have "Read more in the User Guide". Maybe another PR should propose removing all this verbosity.